app/frontend/styles/components/jobs-banner.scss

```diff
@@ -0,0 +1,11 @@
+.jobs-banner {
+  background-color: govuk-colour('light-grey');
+  left: 50%;
```

Comment:
> If position: relative, the left property sets the left edge of an element to a unit to the left/right of its normal position.

Don't the left and right properties cancel out?

Reply:
I think we only need left/margin-left properties and can forgo the right ones